development/file_formats/tscn.rst

```diff
@@ -28,10 +28,12 @@ There are five main sections inside the TSCN file:
 3. Nodes
 4. Connections
 
-The file descriptor looks like ``[gd_scene load_steps=1 format=2]`` and should
-be the first entry in the file. The ``load_steps`` parameter should (in theory)
-be the number of resources within the file. However, in practice, its value seems
-not to matter.
+The file descriptor looks like ``[gd_scene load_steps=3 format=2]`` and should
+be the first entry in the file. The ``load_steps`` parameter is equal to the
+total amount of resources (internal and external) plus one (for the file itself).
+If the file has no resources ``load_steps`` is simply omitted. The engine will
+still load the file correctly if ``load_steps`` is incorrect, but this will affect
+loading bars and any other piece of code relying on that value.
```

Comment:
```suggestion
The file descriptor looks like ``[gd_scene load_steps=3 format=2]`` and should
be the first entry in the file. The ``load_steps`` parameter is equal to the
total amount of resources (internal and external) plus one (for the file itself).
If the file has no resources, ``load_steps`` is omitted. The engine will
still load the file correctly if ``load_steps`` is incorrect, but this will affect
loading bars and any other piece of code relying on that value.
```